### PR TITLE
TL-32739: prebid native global var issue

### PR DESCRIPTION
**Pull request by nllerandi3lift**
Use of global vars to carry the separation of ad units (standard vs native) is causing issues. Moving those to more scoped variables. 
https://triplelift.atlassian.net/browse/TL-32739

### CODE REVIEW

File: modules/tripleliftBidAdapter.js [hunk 1]
```diff
@@ -135,8 +136,8 @@ function _getSyncType(syncOptions) {
 }
 
 function _buildPostBody(bidRequests, bidderRequest) {
-  standardUnits = bidRequests.filter(bid => bid.mediaTypes.banner || bid.mediaTypes.video);
-  nativeUnits = bidRequests.filter(bid => bid.mediaTypes.native && !bid.mediaTypes.banner && !bid.mediaTypes.video);
+  let standardUnits = bidRequests.filter(bid => bid.mediaTypes.banner || bid.mediaTypes.video);
```
Comment (nllerandi3lift) [inline modules/tripleliftBidAdapter.js:139]: Helper function so this more DRY

File: modules/tripleliftBidAdapter.js [hunk 2]
```diff
@@ -94,8 +92,11 @@ export const tripleliftAdapterSpec = {
 
   interpretResponse: function (serverResponse, {bidderRequest}) {
     let bids = serverResponse.body.bids || [];
+    let nativeUnits = bidderRequest.bids.filter(bid => bid.mediaTypes.native && !bid.mediaTypes.banner && !bid.mediaTypes.video);
```
[the remark this answers is not in the document]
Comment (patrickloughrey) [reply modules/tripleliftBidAdapter.js:95]: Added helper function to organize the logic

File: modules/tripleliftBidAdapter.js (hunk 1 above)
Comment (patrickloughrey) [reply modules/tripleliftBidAdapter.js]: Tested helper function. It seems to work in several scenarios. Tests are healthy too

**Review by patrickloughrey — APPROVED** [review]

File: modules/tripleliftBidAdapter.js [hunk 3]
```diff
@@ -94,8 +92,10 @@ export const tripleliftAdapterSpec = {
 
   interpretResponse: function (serverResponse, {bidderRequest}) {
     let bids = serverResponse.body.bids || [];
+    let { standardUnits, nativeUnits } = _splitAdUnits(bidderRequest.bids);
```
Comment (nllerandi3lift) [inline modules/tripleliftBidAdapter.js:95]: a single loop to separate standard from native units

**Pull request merged**